src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,6 +7,95 @@
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<sbyte> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<float> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<string> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<System.Text.Json.JsonNode> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<ushort> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<uint> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<ulong> jsonValues) { }
+        public int Count { get { throw null; } }
+        public bool IsReadOnly { get { throw null; } }
+        public System.Text.Json.JsonNode this[int idx] { get { throw null; } set { } }
+        public void Add(bool jsonValue) { }
+        public void Add(byte jsonValue) { }
+        public void Add(double jsonValue) { }
+        public void Add(short jsonValue) { }
+        public void Add(int jsonValue) { }
+        public void Add(long jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(sbyte jsonValue) { }
+        public void Add(float jsonValue) { }
+        public void Add(string jsonValue) { }
+        public void Add(System.Text.Json.JsonNode jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ushort jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(uint jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ulong jsonValue) { }
+        public void Clear() { }
+        public bool Contains(bool jsonValue) { throw null; }
+        public bool Contains(byte jsonValue) { throw null; }
+        public bool Contains(double jsonValue) { throw null; }
+        public bool Contains(short jsonValue) { throw null; }
+        public bool Contains(int jsonValue) { throw null; }
+        public bool Contains(long jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(sbyte jsonValue) { throw null; }
+        public bool Contains(float jsonValue) { throw null; }
+        public bool Contains(string jsonValue) { throw null; }
+        public bool Contains(System.Text.Json.JsonNode jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(ushort jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(uint jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(ulong jsonValue) { throw null; }
+        public void CopyTo(System.Text.Json.JsonNode[] array, int arrayIndex) { }
```

Comment:
I suppose this just takes an array and copies it starting on arrayIndex? If so, do we want overloads to take arrays for all other types? (bool, byte, double, etc.)

Reply:
Probably it would be consistent to have those.

Reply:
I'm not saying that you have to have those, just consider it to see if you find a scenario where they would be useful. If you think they might, then it's ok to add them, if you don't find a good scenario then it's fine to not have them.

Reply:
OK, I'll investigate it.